docs/advanced.md

```diff
@@ -464,6 +464,16 @@ MIME header field.
     It is safe to upload large files this way. File uploads are streaming by default, meaning that only one chunk will be loaded into memory at a time.
  
  Non-file data fields can be included in the multipart form using by passing them to `data=...`.
+ 
+You can also send multiple files in one go with a multiple file field form.
+To do that you'll need to pass a list of tuples that looks like `(form_field, (2_or_3_elements_tuple))` where the `2_or_3_elements_tuple` is described above and the `form_field` is the name of your form field.
```

Comment:
Little copy suggestion that _might_ be easier to understand?

```markdown
To do that, pass a list of `(field, <file>)` items instead of a dictionary, allowing you to pass multiple items with the same `field`.
```

Reply:
implemented